<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Ico/schedule.ico</normaloff>:/Ico/schedule.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70, 91, 146);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_header">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_error">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 0, 0);
border-radius: 7px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_error">
            <property name="text">
             <string>error</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_error">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-radius: 5px;
	
	background-position: center;
}
QPushButton:hover {
	background-color: rgb(217, 217, 217);
}
QPushButton:pressed {
	background-color: rgb(0, 255, 127);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>close_error.png</normaloff>close_error.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_body">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_content">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>550</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(44, 69, 128);
border-radius: 10px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>376</width>
            <height>134</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>376</width>
            <height>134</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>376</width>
            <height>134</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/Logo_image/logo_login.png);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>logo_login.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_avatar">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>170</y>
            <width>146</width>
            <height>146</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background-color: rgb(44, 69, 128);
background-image: url(:/Login_image/addressbook_1.png);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>addressbook_1.png</pixmap>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLineEdit" name="lineEdit_username">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>330</y>
            <width>280</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(245, 245, 245);
	border-radius: 10px;
	padding: 5px;
	border: 2px solid rgb(143, 59, 59);
}
QLineEdit:hover {
	border: 2px solid rgb(85, 170, 0);
}

</string>
          </property>
          <property name="placeholderText">
           <string>Email or Username</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>380</y>
            <width>280</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(245, 245, 245);
	border-radius: 10px;
	padding: 5px;
	border: 2px solid rgb(143, 59, 59);
}
QLineEdit:hover {
	border: 2px solid rgb(85, 170, 0);
}
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_access">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>440</y>
            <width>280</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(102, 178, 247);
	border-radius: 5px; 
}
QPushButton:hover {
	background-color: rgb(44, 69, 128);
}
QPushButton:pressed {
	background-color: rgb(0, 255, 127);
}</string>
          </property>
          <property name="text">
           <string>Acessar</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_footer">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 69, 128);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Criado por CCN Serviços em TI</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="file_rc.qrc"/>
 </resources>
 <connections/>
</ui>
